book-edited.tex

```diff
@@ -5930,7 +5930,7 @@ \section{Outline}\label{LayingOutPages-outline}}
 \section{Exercises}\label{LayingOutPages-exercises}}
 \begin{enumerate}[label=\thechapter-\arabic*]
 \item \emph{Links bar.} At the top and bottom of the web version of
-  each chapter of this book there is
+  each chapter of this book is
```

Comment:
I prefer the “there”, was the copy editor against?

Reply:
It wasn't in their copy.